package err

func F() {
L1:
	continue L1
}
